readme.md

```diff
@@ -237,11 +237,13 @@ type ShouldBeNever = IfAny<'not any', 'not never', 'never'>;
 - [`Trim`](source/trim.d.ts) - Remove leading and trailing spaces from a string.
 - [`Split`](source/split.d.ts) - Represents an array of strings split using a given character or character set.
 - [`Replace`](source/replace.d.ts) - Represents a string with some or all matches replaced by a replacement.
+- [`StringSlice`](source/string.d.ts) - Returns a string slice of a given range(with negative index support), just like `String.slice` in js.
```

Comment:
```suggestion
- [`StringSlice`](source/string.d.ts) - Returns a string slice of a given range(with negative index support), just like `String#slice` in js.
```

Reply:
> with negative index support

This is not important enough to have here. can be in the file doc comment instead.

Reply:
updated